app/grandchallenge/archives/views.py

```diff
@@ -558,6 +564,115 @@ def get_serializer_class(self):
             return ArchiveItemSerializer
 
 
+class ArchiveItemJobListView(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    JobsList,
+):
+    model = Job
+    template_name = "archives/archive_item_job_list.html"
+    row_template = "algorithms/job_list_row.html"
+
+    search_fields = [
+        *JobsList.search_fields,
+        # Outputs
+        "outputs__interface__title",
+        "outputs__interface__description",
+        # Algorithm
+        "algorithm_image__algorithm__title",
+        "algorithm_image__algorithm__description",
+        "algorithm_image__comment",
+    ]
+
+    permission_required = (
+        f"{Archive._meta.app_label}.view_{ArchiveItem._meta.model_name}"
+    )
+    raise_exception = True
+
+    login_url = reverse_lazy("account_login")
+
+    @cached_property
+    def archive_item(self):
+        return get_object_or_404(ArchiveItem, pk=self.kwargs["pk"])
+
+    def get_permission_object(self):
+        return self.archive_item
+
+    def get_queryset(self):
+        input_civs = self.archive_item.values.all()
+        num_civs = len(input_civs)
+
+        jobs_subquery = Job.objects.filter(inputs__in=input_civs).distinct()
+
+        query_set = (
+            Job.objects.filter(pk__in=jobs_subquery)
+            .annotate(
+                num_inputs=Count("inputs"),
+            )
+            .filter(num_inputs=num_civs)
+            .annotate(
+                inputs_match_count=Count(
+                    "inputs",
+                    filter=Q(inputs__in=input_civs),
+                )
+            )
+            .filter(inputs_match_count=num_civs)
+            .prefetch_related(
+                "outputs__image__files",
+                "outputs__interface",
+                "inputs__image__files",
+                "inputs__interface",
+                "viewers__user_set",
+            )
+            .select_related(
+                "creator__user_profile",
+                "creator__verification",
+                "algorithm_image__algorithm",
+            )
+        )
+
+        return filter_by_permission(
+            queryset=query_set,
+            user=self.request.user,
+            codename="view_job",
+            accept_user_perms=False,
+        )
+
+    def get_row_context(self, *, object_, **kwargs):
+        context = super().get_row_context(object_=object_, **kwargs)
+        context.update(
+            {
+                "algorithm": object_.algorithm_image.algorithm,
+            }
+        )
+        return context
+
+    def get_context_data(self, *args, **kwargs):
+        context = PaginatedTableListView.get_context_data(
+            self, *args, **kwargs
+        )
```

Comment:
This should be `super().get_context_data(*args, **kwargs)`

Reply:
Done!